symengine/constants.cpp

```diff
@@ -46,6 +46,7 @@ RCP<const Number> I = Complex::from_two_nums(*zero, *one);
 
 RCP<const Constant> pi = constant("pi");
 RCP<const Constant> E = constant("E");
+RCP<const Constant> euler_const = constant("euler_const");
```

Comment:
Name of this constant should be changed. Maybe `EulerGamma` as in SymPy and Mathematica or `euler_gamma` as in Sage